reactivesocket/src/test/scala/akka/stream/alpakka/reactivesocket/scaladsl/ReactiveSocketSpec.scala

```diff
@@ -0,0 +1,94 @@
+package akka.stream.alpakka.reactivesocket.scaladsl
+
+import akka.stream.scaladsl._
+import io.reactivesocket.util.PayloadImpl
+import org.scalatest.WordSpec
+
+import scala.concurrent.duration._
+
+class ReactiveSocketSpec extends WordSpec {
+
+  "reactive socket API" should {
+
+    "support server and client" in {
+
+      import io.reactivesocket.transport.tcp.client.TcpTransportClient
+      import io.reactivesocket.transport.tcp.server.TcpTransportServer
+
+      val binding = ReactiveSocketServer(TcpTransportServer.create())
+          .toMat(Sink.foreach {
+            _.flow.runWith(Source.repeat(new PayloadImpl("Pong")), Sink.foreach(println))
+          })(Keep.left)
+          .run()
+
+      implicit val rs = ReactiveSocket(TcpTransportClient.create(binding.localAddress))
+
+      ReactiveSocketClient.stream(new PayloadImpl("One time ping")).runForeach(println)
+
+      Source.tick(1.second, 1.second, new PayloadImpl("Continuous ping")).via(ReactiveSocketClient.channel()).runForeach(println)
+
+    }
+
+    "support local transport" in {
+
+      import io.reactivesocket.local.LocalServer
+      import io.reactivesocket.local.LocalClient
+
+      val address = "test-local-server"
+
+      val binding: Source[ReactiveSocketServer.IncomingConnection, ReactiveSocketServer.Binding] = ReactiveSocketServer(LocalServer.create(address))
+
+      implicit val rs = ReactiveSocket(LocalClient.create(address))
+
+      Source.tick(1.second, 1.second, new PayloadImpl("Hi")).via(ReactiveSocketClient.channel()).runForeach(println)
+    }
+
+    "support aeron transport" in {
+
+      import io.reactivesocket.aeron.internal._
+      import io.reactivesocket.aeron.internal.reactivestreams._
+      import io.reactivesocket.aeron.server.AeronTransportServer
+      import io.reactivesocket.aeron.client.AeronTransportClient
+
+      // server
+
+      // aeron config, which should be hidden from the user
```

Comment:
We should hide the reactive socket library, we should provide a more convenient api for integration with Akka things. I don't think we need to provide anything for the transport creation.